README.md

```diff
@@ -20,34 +20,34 @@ The possibilities are endless.
 
 ### Building the Application
 
-Now that you have not done anything it's time to build your application:
+Now that nothing has been done, it's time to build the application:
 
 ```
 
 ```
 
-Yep. That's it. You should see the following output:
+Yep. That's it. The following output should be seen:
 
 ```
 
 ```
 
 ### Deploying
 
-While you still have not done anything it's time to deploy your application. By running the following command you can deploy your application absolutely nowhere.
+While nothing still has not been done, it's time to deploy the application. By running the following command the application can be deployed absolutely nowhere.
 
 ```
 
 ```
 
-It's that simple. And when it comes time to scale the application, all you have to do is:
+It's that simple. And when it comes time to scale the application, all that has to be done:
 
 ```
 
 ```
 
-I know right?
+"I know right?"
```

Comment:
There should be a comma here, following the 'know'.
ie. `"I know, right?"`

Else, this sentence is much more philosophical in nature.
Questioning whether one knows either the relative direction or a subset of acceptable human behaviour is most likely not the intention here.